packages/openapi-2-kong/src/common.js

```diff
@@ -68,7 +68,7 @@ export function generateSlug(str: string, options: SlugifyOptions = {}): string
 const pathVariableSearchValue = /{([^}]+)}(?!:\/\/)/g;
 
 export function pathVariablesToRegex(p: string): string {
-  const result = p.replace(pathVariableSearchValue, '(?<$1>\\S+)');
+  const result = p.replace(pathVariableSearchValue, '(?<$1>[^\\/\\s]+)');
```

Comment:
It would be worth adding a comment here to explain this regex, something like

```suggestion
  // match any non-whitespace character, excluding /
  const result = p.replace(pathVariableSearchValue, '(?<$1>[^\\/\\s]+)');
```